src/lib/Filter/VersionFilter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace EzSystems\HybridPlatformUi\Filter;
+
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+
+class VersionFilter
+{
+    public function filterDrafts(array $versions)
```

Comment:
Good point, I was looking at some of the other classes in this library and they are missing too which is why it was missed. For example `EzSystems\HybridPlatformUi\EventSubscriber\AppRendererSubscriber`
`EzSystems\HybridPlatformUi\EventSubscriber\ComponentRendererSubscriber`
I shall add it in this class and add some documentation for the methods 😄